src/main/java/frc/robot/commands/finishScore.java

```diff
@@ -1,7 +1,24 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Cal;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lift.LiftPosition;
 
-public class finishScore extends CommandBase {
-  // TODO do this
-}
+public class finishScore extends SequentialCommandGroup {
+  public finishScore(Lift lift){
+    addRequirements(lift);
+    addCommands(
+      new InstantCommand(lift::openGrabber, lift),
+      new WaitCommand(Cal.Lift.GRABBER_OPEN_TIME_SECONDS),
+      (lift.isScoringHigh() ?
+        new InstantCommand(() -> lift.setDesiredPosition(LiftPosition.POST_SCORE_HIGH), lift).andThen(
+          new WaitUntilCommand(() -> lift.atPosition(LiftPosition.POST_SCORE_HIGH))) : 
+        null),
+      new InstantCommand(() -> lift.setDesiredPosition(LiftPosition.STARTING))
```

Comment:
Maybe we should just wait an arbitrary amount of time here instead of waiting for it to get all the way to this position